modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -234,9 +249,11 @@ It is strongly recommend to use a reverse proxy like {traefik-url}[Traefik], {ng
 
 == Filesystems and Shared Storage
 
-In addition to well known metadata like _name_, _size_ and _mtime_ (time a file was last modified), oCIS allows users to add arbitrary metadata like _tags_ and _comments_. Therefore oCIS requires and works with POSIX-compliant file systems where this metadata can be mapped to extended attributes. S3 storage is supported too, but requires POSIX-compliant local storage for metadata persistance.
+oCIS handles well known metadata like _name_, _size_ and _mtime_ (time a file was last modified). Therefore it requires POSIX-compliant file systems where this metadata can be mapped to extended attributes. 
+
+NOTE: An S3 compatible object storage may be supported in the future, but this will require a an extra POSIX-compliant local storage for metadata persistance.
```

Comment:
```suggestion
NOTE: An S3 compatible object storage may be supported in the future, but this will require an extra POSIX-compliant local storage for metadata persistance.
```